lib/rubocop/config_loader.rb

```diff
@@ -55,7 +55,7 @@ def load_file(file)
       end
 
       def add_missing_namespaces(path, hash)
-        hash.keys.each do |key|
+        hash.keys.each do |key| # rubocop:disable Style/HashEachMethods
```

Comment:
I guess we can just update this, right?

Reply:
I actually don't think we can in this case. Due to the `hash[q] = hash.delete(key)` line below, we'll get a `     RuntimeError: can't add a new key into hash during iteration` if we try do a `hash.each_key` here.

Reply:
Ops. my bad. Should have read the code more carefully. :-)

Reply:
Could this whole issue be sidestepped with a
```ruby
hash.transform_keys { |key| Cop::Cop.qualified_cop_name(key, path) }
```
? Or do we need to support rubies < 2.5?